### ISSUE TITLE
microwaves are horribly broken

### ISSUE THREAD

**Issue by deltanedas** [space-wizards/space-station-14#14874]
## Description
- anything not cooked is not ejected
- food is not ejected when unpowered
- **can work without power!!!**
- you have to insert an item then take it out to eject unejected items

**Comment by jamessimo** [comment]
I am working on UI fixed to the microwave and I will see if I can tackle any of these. 

Also wanted to bring the conversation over here. On Discord we were talking about what the instant cook button does and figure out it’s  the exact same thing as pressing five seconds. I’m gonna get rid of it in the next version since it’s only there (as far as we can tell) because of needing a 0 time option to satiate a for loop.

**Comment by deltanedas** [comment]
thats wrong, i think its just a bug that in the ui instant is default but 5 seconds is the actual default
switching to 5 then instant actually instantly cooks it

**Comment by potato1234x** [comment]
the instant button was useful for cooking an id instantly

**Comment by Krzeszny** [comment]
Tested, same 4 problems as mentioned in OP. 

Tbh, this issue is 2 individual issues:
A) the microwave works without power as long as it's anchored
B) the microwave doesn't eject uncooked items

How to repro: 
1. Anchor the microwave somewhere outside of the range of LV cables
2. Place dough in it
3. Use INSTANT or 5 seconds mode
4. Start
5. The microwave starts cooking but locks itself.

Workaround: unanchor or place more ingredients inside to eject.

Expected:
- microwave works only when powered
- uncooked items get ejected
